integration/chirpstack.md

```diff
@@ -30,12 +30,13 @@ On the Chirpstack-UI, open your application and then click on the *Integrations*
 * **Endpoint**: Add `https://integrations.ttnmapper.org/chirp/v3/events`
 
 ### Headers
-#### Required
+
+#### Mandatory
 * **TTNMAPPERORG-NETWORK**: Name of your network, for example: `my.network.name`. Try and make this unique, as this value will be used to distinguish your network's coverage and gateways from other ChirpStack networks' coverage and gateways. This header is a temporary workaround until a proper network identifier is added to ChirpStack. See [this Github issue](https://github.com/brocaar/chirpstack-network-server/issues/532)
+* **TTNMAPPERORG-USER**: Add your email address in this header to allow TTN Mapper to associate your coverage data with you. This will allow to easily find your data, and also verify when you request your data to be deleted.
```

Comment:
Please revert this change. Because ChirpStacks are inherently private networks, we do not enforce identifying the data like we do on public networks like TTN an Helium.